util/analyze-contention-log.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/sh
+
```

Comment:
Should we set some options like `set -e -o pipefail` here?

https://linuxtect.com/make-bash-shell-safe-with-set-euxo-pipefail/